themes/default/content/docs/concepts/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -35,7 +35,9 @@ Both Pulumi and CDK allow you to build and deploy infrastructure on AWS using fa
 
 ## Pulumi vs. CDK: Key Differences {#differences}
 
-First, CDK supports only AWS, whereas Pulumi supports over 150 cloud and SaaS providers, with more being added all the time. Second, while both Pulumi and CDK support languages like TypeScript, Python, Go, C#, and Java, the two tools are fundamentally different in how they interpret these languages and deploy resources. Unlike Pulumi, whose open-source engine understands these languages and communicates directly with cloud providers to deploy infrastructure, CDK is a transpiler (i.e., a [source-to-source compiler](https://en.wikipedia.org/wiki/Source-to-source_compiler)) that produces AWS Cloud Assembly, an intermediate format consisting of CloudFormation templates and other assets. After the assets are uploaded to the appropriate services (S3, ECR), CDK uses the CloudFormation service, which acts as the deployment engine for provisioning resources.
+First, CDK supports only AWS, whereas Pulumi supports over 150 cloud and SaaS providers, with more being added all the time. Second, while both Pulumi and CDK support languages like TypeScript, Python, Go, C#, and Java, the two tools are fundamentally different in how they interpret these languages and deploy resources. Unlike Pulumi, whose open-source engine understands these languages and communicates directly with cloud providers to deploy infrastructure, CDK is a transpiler (i.e., a [source-to-source compiler](https://en.wikipedia.org/wiki/Source-to-source_compiler)) that first produces AWS Cloud Assembly, an intermediate format consisting of CloudFormation templates and other assets. After the transpilation process, these assets are uploaded to targeted services (such as S3, ECR) and then the CDK invokes the CloudFormation service, which begins the deployment process.
+
+The limits of the transpiler present some significant problems in terms of both speed (of both the inner development loop and the actual deployment) and correctness (since errors may not be discovered until hours into the deployment process, rather than at compile time.  Because of the inherent CDK dependencies, users need expertise in both CloudFormation and the CDK in order to debug any issues and achieve any successful results.
```

Comment:
```suggestion
The limits of the transpiler present some significant problems in terms of both speed (of both the inner development loop and the actual deployment) and correctness (since errors may not be discovered until hours into the deployment process, rather than at compile time. Because of the inherent CDK dependencies, users need expertise in both CloudFormation and the CDK in order to debug any issues and achieve any successful results.
```